code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -350,7 +350,7 @@
 	if(C.suiciding)
 		return FALSE //Kevorkian school of robotic medical assistants.
 
-	if(ishuman(C))
+	if(ishuman(C) && !(emagged == 2))

```

Comment:
Just move this block under the emagged check below it.

Reply:
that works